util/build_configs/README.md

```diff
@@ -27,6 +27,46 @@ Work-in-progress (2018-07-31)
   logs, CHPL_HOME workspaces, etc in this directory, without adding
   them to Git.
 
+  Let's say your host machine has Slurm; you want a Chapel multi-config
+  build with and without Gasnet (where Gasnet may use either of two widely-
+  supported comm layers); and you also want the option to use the native
+  Slurm launcher slurm-srun as well as Chapel's default Gasnet launcher(s).
```

Comment:
What do you mean by `Chapel's default Gasnet launcher(s).` here? Are you referring to `CHPL_COMM_SUBSTRATE` values?

Reply:
From https://chapel-lang.org/docs/usingchapel/launcher.html,

> 
> otherwise, if CHPL_COMM is gasnet:
> 
> If | CHPL_LAUNCHER
> -- | --
> CHPL_COMM_SUBSTRATE=ibv | gasnetrun_ibv
> CHPL_COMM_SUBSTRATE=mpi | gasnetrun_mpi
> CHPL_COMM_SUBSTRATE=mxm | gasnetrun_ibv
> CHPL_COMM_SUBSTRATE=ofi | gasnetrun_ofi
> CHPL_COMM_SUBSTRATE=psm | gasnetrun_psm
> CHPL_COMM_SUBSTRATE=smp | smp
> CHPL_COMM_SUBSTRATE=udp | amudprun
> otherwise | none
>

Reply:
OK, makes sense.